compiler/rustc_session/src/config/cfg.rs

```diff
@@ -142,6 +142,7 @@ pub(crate) fn disallow_cfgs(sess: &Session, user_cfgs: &Cfg) {
             | (sym::target_has_atomic, Some(_))
             | (sym::target_has_atomic_equal_alignment, Some(_))
             | (sym::target_has_atomic_load_store, Some(_))
+            | (sym::target_has_reliable_f16_f128, Some(_))
```

Comment:
I don't see the updated documentation, nor the feature gating of the cfg.

The steps to do are roughly described at the top of this file.

Reply:
I think it is being gated correctly - isn't this covered by GATED_CFGS in builtin_attrs.rs? The tests/ui/feature-gates test should also be checking this.

I've updated the check-cfg list. Do internal features also get documented under `language-features` in the unstable book?

Reply:
My bad, didn't see the gating on `builtin_attrs.rs` (I should learn to not do reviews on my phone).